tests/uefistored/conftest.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+
+def imported_sb_vm(imported_vm):
+    vm = imported_vm
+
+    tmp_pk = vm.host.ssh(['mktemp'])
+    tmp_kek = vm.host.ssh(['mktemp'])
+    tmp_db = vm.host.ssh(['mktemp'])
+    tmp_dbx = vm.host.ssh(['mktemp'])
+
+    vm.host.ssh(['mv', '/usr/share/uefistored/PK.auth', tmp_pk], check=False)
+    vm.host.ssh(['mv', '/usr/share/uefistored/db.auth', tmp_db], check=False)
+    vm.host.ssh(['mv', '/usr/share/uefistored/KEK.auth', tmp_kek], check=False)
+    vm.host.ssh(['mv', '/usr/share/uefistored/dbx.auth', tmp_dbx], check=False)
+    vm.host.ssh(['varstore-sb-state', vm.uuid, 'setup'])
+
+    yield vm
+
+    # Restore PK, KEK, db on host
+    vm.host.ssh(['mv', tmp_pk, '/usr/share/uefistored/PK.auth'], check=False)
+    vm.host.ssh(['mv', tmp_kek, '/usr/share/uefistored/KEK.auth'], check=False)
+    vm.host.ssh(['mv', tmp_db, '/usr/share/uefistored/db.auth'], check=False)
+    vm.host.ssh(['mv', tmp_dbx, '/usr/share/uefistored/dbx.auth'], check=False)
+
+@pytest.fixture(scope='module')
+def windows_sb_vm(imported_vm):
+    vm = imported_vm
+
+    if not vm.is_windows:
+        pytest.skip('only valid for Windows VMs')
+
+    yield from imported_sb_vm(vm)
```

Comment:
I didn't know we could use `yield from` in fixtures. I suppose the teardown starting at line 20 does get executed by pytest?